### ISSUE TITLE
Should Row and Column have width and height parameters?

### ISSUE THREAD

**Issue by jbednar** [pyviz/panel#23]
Right now, Panes have width and height (most of them), but it looks like rows and columns only adapt to the size of the items within them.  Can Row and Column support width and height (defaulting to None) so that people can reserve space for a variable number and size of thing that might be in them?

Of course, we'd immediately want control over what happens to the things inside relative to the space reserved (i.e. centered or justified in either direction, for both x and y), but we probably already need that anyway just because not everything in the same row or column will be the same width.

**Comment by philippjfr** [comment]
>Right now, Panes have width and height (most of them), but it looks like rows and columns only adapt to the size of the items within them. Can Row and Column support width and height (defaulting to None) so that people can reserve space for a variable number and size of thing that might be in them?

No, at least currently bokeh Row and Column models do not have a width or height.

**Comment by jbednar** [comment]
Do you know how people usually handle that in Bokeh (making a minimum-sized region in which other things can be placed without affecting surrounding items)?

**Comment by philippjfr** [comment]
You can use a ``Spacer``, which is also supported in HoloViews.

**Comment by philippjfr** [comment]
Once the new layout work is merged into bokeh, Row/Column **will** have a width and height.

**Comment by kcpevey** [comment]
Great! I'm glad to see this implemented.

**Comment by philippjfr** [comment]
Don't want to overpromise, I don't yet know what the semantics of Row/Column width and height settings will be, i.e. if it actually forces subplots to conform to the size, or just cuts them off or whatever.

**Comment by philippjfr** [comment]
This is now supported.
